react-app-drawer-component/src/app-drawer.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+class AppDrawer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      drawerOpen: false,
+      showMenu: true
+    };
+    this.handleClickDrawer = this.handleClickDrawer.bind(this);
+  }
+
+  handleClickDrawer() {
+    this.setState({
+      drawerOpen: !this.state.drawerOpen,
+      showMenu: !this.state.showMenu
```

Comment:
No I mean only one property instead of both. 😂

Reply:
Ohh my bad haha. Will fix!